devops/test_configs.json

```
@@ -33,6 +33,17 @@
       "check_sycl_all": "opencl:cpu,host",
       "cmake_args": ""
     },
+    {
+      "config": "esimd_emu",
+      "name": "ESIMD Emu LLVM Test Suite",
+      "runs-on": [
+        "x64"
+      ],
+      "image": "${{ inputs.intel_drivers_image }}",
+      "container_options": "-u 1001",
+      "check_sycl_all": "ext_intel_esimd_emulator:gpu,host",
```

```suggestion
      "check_sycl_all": "ext_intel_esimd_emulator:gpu",
```
Can we exclude host device from this run?